### ISSUE TITLE
Add a Wordle command

### ISSUE THREAD

**Issue by dannynotsmart** [python-discord/sir-lancebot#1034]
## Description
Wordle is a very, very popular word game that's rising in players.

## Reasoning
It's a game that would be fun to play with others, and the game would probably appeal to most people. 

## Proposed Implementation
I have built a working Wordle generator, which would be too complicated to go in detail on Github. It will use modals, buttons. 
There are only two issues currently, and that's 1. Modals aren't supported by discord.py, and I do not see SirLancebot using an alternative. However, I also have a solution for this.
2. Where would I get a 5 lettered word? I'm thinking API or a text file

## Additional Details
Please ping me on Discord if you would like to discuss about this, The Cat#8308

## Would you like to implement this yourself?

- [x] I'd like to implement this feature myself
- [x] Anyone can implement this feature

**Comment by Feroz2006** [comment]
I have a working solution to get a random 5-letter word from an API (in agreement with their terms of service) however I can't seem to find you using the discord ID provided.

**Comment by DMFriends** [comment]
Yeah I can't seem to find them either on PyDis. Have you tried sending them a friend request, @Feroz2006?

**Comment by Feroz2006** [comment]
The request fails with the given ID, must've changed their ID and forgot to update the issue.

**Comment by parth-verma7** [comment]
@dannynotsmart I would like to work upon this issue. I got a 5 letter word txt file and I would like to accomplish the task2 for now. Please assign it to me. Thank You.
